components/yaml/yaml_format.rst

```diff
@@ -171,8 +166,8 @@ Collections
 -----------
 
 A YAML file is rarely used to describe a simple scalar. Most of the time, it
-describes a collection. A collection can be a sequence or a mapping of
-elements. Both sequences and mappings are converted to PHP arrays.
+describes a collection. YAML collections can be a sequence (scalar arrays in PHP)
```

Comment:
Not sure what you mean by `scalar arrays`. Perhaps `numerically-indexed arrays` would be better?